docs/Remote-zqd.md

```diff
@@ -0,0 +1,310 @@
+new data as soon as it's added to a remote location. The steps below provide
+guidance on how to "refresh" the Brim interface to ensure all remote data is
+visible.
+
+# Background: Brim & `zqd`
+
+Since it's presented as an icon that can be double-clicked to launch it on
+your desktop, it's easy to think of Brim as a simple standalone application.
+However, it actually includes some "backend" components that assist in
+providing the overall app experience.
+
+One essential component is [`zqd`](https://github.com/brimsec/zq/tree/master/ppl/cmd/zqd),
+a server-style process that manages the storage and querying of imported
+log/packet data.  Operations in `zqd` are invoked via a
+[REST API](https://en.wikipedia.org/wiki/Representational_state_transfer)
+that's utilized by a "client", such as the Brim app and/or the
+[`zapi`](https://github.com/brimsec/zq/tree/master/cmd/zapi) command line tool.
+
+![Brim zapi and zqd](media/Brim-zapi-zqd.png)
+
+The location where `zqd` stores imported data is known as the
+**Data Directory**. This location can be changed via a setting in Brim's
+**Preferences** menu. The default Data Directory locations on each platform
+are as follows:
+
+|**OS Platform**|**Location**|
+|---------------|------------|
+| **Windows**   | `%APPDATA%\Brim\data\spaces`                         |
+| **macOS**     | `$HOME/Library/Application Support/Brim/data/spaces` |
+| **Linux**     | `$HOME/.config/Brim/data/spaces`                     |
+
+If you examine the process table while Brim is running, you can observe the
+command line that was used to start the backend `zqd` process. For example,
+here is the process on a Mac laptop being operated by username "phil".
+
+```
+macOS# ps auxww | grep zqd
+phil             29475 180.6  0.2  5042684  32340   ??  R     4:38PM   0:01.16 /Applications/Brim.app/Contents/Resources/app/zdeps/zqd listen -l localhost:9867 -data /Users/phil/Library/Application Support/Brim/data/spaces -config /Users/phil/Library/Application Support/Brim/zqd-config.yaml -zeekrunner /Applications/Brim.app/Contents/Resources/app/zdeps/zeek/zeekrunner -brimfd=3
+```
+
+Some useful information revealed in this command line:
+
+1. The presence of the `listen` command indicates that the `zqd` process is
+prepared to accept REST API requests.
+
+2. The inclusion of `localhost` in the option `-l localhost:9867` indicates
+this `zqd` is prepared to accept _only_ connections that arrive from a client
+running on the same local host.
+
+3. The `-data` option points to the Data Directory, which is the default
+location for macOS in this case.
+
+4. The `-zeekrunner` option points to a script that is used to initiate the
+creation of Zeek logs from imported packet captures as described in the
+[Zeek Customization](https://github.com/brimsec/brim/wiki/Zeek-Customization)
+article.
+
+5. The `-brimfd=3` is an option unique to when `zqd` is launched by Brim.
```

Comment:
I don't think we need to mention this.

Reply:
@mattnibs: I'd love to not have to mention it. The problem is that further down in the article I need to walk the user through creating a `zqd` command line that drops it. So once I show it being removed, it calls into question why it's there and why it had to be removed in order for the command line to work. That's why I explain what it is where it first appears.

I think you're picking up on how this kind of article is a balancing act. Many of us have been on-board for some time thinking users might benefit from starting to use remote `zqd` configs. However all along the way I've encountered bugs or usability issues that would make it difficult for all but the most experienced users. This is why I put the qualifying info in "About Cookbooks" saying the stuff is new/experimental and that exposing some of the innards is part of the point. If we had the dev/UX resources to address the things I still had to cite in "Limitations" and other usability improvements I didn't even mention, this could be more of a "For Dummies" thing where we'd not have to talk innards. Sadly, I don't think we're there quite yet, so I went this route in the interest of getting something out there rather than waiting on perfection.